core/chains/evm/types/types.go

```diff
@@ -53,6 +55,23 @@ type Receipt struct {
 	TransactionIndex  uint            `json:"transactionIndex"`
 }
 
+type EvmFee struct {
```

Comment:
```suggestion
type EVMFee struct {
```

Reply:
It seems like it's not being used anywhere. I'll remove it instead.